pandas/tests/reshape/test_pivot.py

```diff
@@ -2549,6 +2549,53 @@ def test_crosstab_tuple_name(self, names):
         result = pd.crosstab(s1, s2)
         tm.assert_frame_equal(result, expected)
 
+    @pytest.mark.parametrize(
+        "s1_data, s1_name, s2_data, s2_name, "
+        "expected_index, expected_column, expected_data",
```

Comment:
what is the unique feature between these sets of parameters you are trying to test? Wasn't immediately clear to me; may be OK to do without it if it simplifies test

Reply:
yeah, removed. the unique feature was different shape of output, and with one of `index` or `column` having same value, like `[1,1,1]`, but probably made it unclear. I now just used the example described in the issue